### PR TITLE
TFD-11505 - AmazonServiceException in logs from INFO to DEBUG

### PR DESCRIPTION
**Pull request by gmendy1**

### CODE REVIEW

File: hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java [hunk 1]
```diff
@@ -1473,12 +1473,12 @@ public long getDefaultBlockSize() {
   }
 
   private void printAmazonServiceException(AmazonServiceException ase) {
```
Comment (amarouni) [inline hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java:1475]: What do you think about adding a warning or info message informing the user that an exception has been thrown and they can get more information by setting debug mode ?

**Review by amarouni — APPROVED** [review]
LGTM

File: hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java (hunk 1 above)
Comment (gmendy1) [reply hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java:1475]: OK but the Exception is re-thrown anyway, so the the user of this fonction always have the opportunity to print it if needed.

Comment (amarouni) [reply hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java:1475]: Fair enough !
Let's keep it this way

**Pull request merged**